src/modules/sub-items/sub.items.module.js

```diff
@@ -896,7 +1246,7 @@ SubItemsModule.propTypes = {
     generateLink: PropTypes.func.isRequired,
     totalCount: PropTypes.number,
     languages: PropTypes.object,
-    udwConfigBulkMoveItems: PropTypes.object.isRequired,
+    udwConfigBulkItems: PropTypes.object.isRequired,
```

Comment:
You can't use the same UDW config for both `move` and `add`. 
The `udwConfigBulkMoveItems` should stay and you should add new config `udwConfigBulkAddLocation` and pass it in the admin-ui bundle. 

Here should be set (`data-udw-config-bulk-add-location="{{ ez_udw_config('single_container', {}) }}"`) https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/themes/admin/content/location_view.html.twig#L88 and here should be passed to the subitems https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/public/js/scripts/admin.location.view.js#L167

Reply:
Thanks @dew326 , good catch
Is it okay to make `udwConfigBulkAddLocation` required? Or, may I make it optional and use config from `udwConfigBulkMoveItems` when not defined?

Reply:
It should be required. Those are two different configs and we cannot do a fallback one on the other. 
It's ok to add required config as this will be part of 3.0.

Reply:
Okay, done, please review.
I will work on admin-ui PR